README.md

```diff
@@ -1 +1,43 @@
-# youtube-simulator-page
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# YouTube Webpage
+> Task is to clone YouTube’s video player page using web archive link: http://archive.today/Bss88 as a source page.
+
+We made a clone of the youtube player page, it has a functional video, buttons, and interactive icons. The page works best on a 1366x768 resolution and using the firefox browser, assets might look out of place using other browsers or resolutions. It has mock suggestions showing video name and views, and a mock section for commentaries. The focus of this branch was to make everything fit in their right places, along with a functional youtube video player.
+
+![screenshot](./File/img/demopage.png)
+ 
+## Built With
+
+- HTML5
+- CSS
+- Font Awesome
+
+## Live Demo
+
+[Live Demo Link](https://railona.github.io/youtube-simulator-page/)
+
+👤 **Railon Acosta**
+
+- GitHub: [@railonA](https://github.com/RailonA)
```

Comment:
If you have more social links like `linkedin` or `twitter` I would suggest you to add them here as well